_additional_info.qmd

```diff
@@ -0,0 +1,19 @@
+# **Additional Information**
+***
+
+<!--
+Use this section to provide any additional information or resources on the topic and methods covered in the case study
+-->
+
+## **Helpful Links**
+***
+
+<u>Terms and concepts covered:</u>
+
```

Comment:
great question, I have been thinking about simplifying the package section or making it collapsable for situations where they just need to load the tidyverse - instead of saying all the tidyverse packages used if that makes sense, certainly we will be using more packages though. I think we could play around with this and see what it looks with the automated form